### PR TITLE
README, User Guide Cleanup

### PR DESCRIPTION
**Pull request by dseah-intel**

### CODE REVIEW

File: Docs/svt-hevc_encoder_user_guide.md [hunk 1]
```diff
@@ -342,9 +342,14 @@ Visual Studio 2017 offers Profile Guided Optimization (PGO) to improve compiler
...
 Some Linux\* Operating systems and kernels assign CPU utilization limits to applications running on servers. Therefore, to allow the application to utilize up to ~100% of the CPUs assigned to it, it is best to run the following commands before and when running the encoder:
 
->sudo  sysctl  -w  kernel.sched\_rt\_runtime\_us=1000000
+> sudo  sysctl  -w  kernel.sched\_rt\_runtime\_us=1000000
 - this command should be executed every time the server is rebooted
 
+> ./run_encode &
+  export PID=$! 
+  sudo chrt -f -a -p 99 $PIDGME
+- this command should be executed with "-rt 1" with the SVT-HEVC application to allow run-time priorities
```
Comment (tianjunwork) [inline Docs/svt-hevc_encoder_user_guide.md]: what is ./run_encoder? is $PIDGME a typo?
run encoder as background is the only way to set real-time?

Comment (dseah-intel) [reply Docs/svt-hevc_encoder_user_guide.md]: Should I use "SvtHevcEncApp -i input.mp4 -w 3840 -h 2160" instead?
$PIDGME is a typo, will change promptly.

Comment (tianjunwork) [reply Docs/svt-hevc_encoder_user_guide.md]: I think "SvtHevcEncApp -i input.**yuv** -w 3840 -h 2160" is more clearer to reader.

File: Source/Lib/Codec/EbEncHandle.c [hunk 2]
```diff
@@ -564,6 +564,8 @@ void SwitchToRealTime()
     };
 
     pthread_setschedparam(pthread_self(), SCHED_FIFO, &schedParam);
+    if (retValue == EPERM)
+        SVT_LOG("\n[WARNING] Elevated privileges required to run with real-time policies! Check Linux Best Known Configuration in User Guide to run application in real-time without elevated privileges!\n\n");
```
Comment (tianjunwork) [inline Source/Lib/Codec/EbEncHandle.c]: retValue is not defined?

File: Docs/svt-hevc_encoder_user_guide.md (hunk 1 above)
Comment (dseah-intel) [reply Docs/svt-hevc_encoder_user_guide.md]: Fixed

File: Source/Lib/Codec/EbEncHandle.c (hunk 2 above)
Comment (dseah-intel) [reply Source/Lib/Codec/EbEncHandle.c]: Fixed

**Pull request merged**